app/helpers/record_helper.rb

```diff
@@ -86,7 +86,8 @@ def gis_access_link(metadata)
     if access_type(metadata) == 'unknown: check with owning institution'
       links.select { |link| link['kind'] == 'Website' }.first['url']
     else
-      links.select { |link| link['kind'] == 'Download' && link['text'] == 'Data' }.first['url']
+      url = links.select { |link| link['kind'] == 'Download' && link['text'] == 'Data' }.first['url']
+      "#{url}?timdexui=true"
```

Comment:
My only question is whether there's going to be any of these links that already have a querystring in them, which would result in something like

```
https://cdn.dev1.mitlibrary.net/geo/public/US_MA_P3STATIONS_2004.zip?foo=bar?timdexui=true
```